docs/guides/Guide-Intro.md

```diff
@@ -56,6 +56,8 @@ const SimpleApp = StackNavigator({
 });
 
 AppRegistry.registerComponent('SimpleApp', () => SimpleApp);
+
+export default SimpleApp;
```

Comment:
can you move it above the AppRegistry?

Reply:
It should work. I'll update the pull request.